### PR TITLE
OCD-3538: Archive audit data

### PR DESCRIPTION
**Pull request by tmy1313**

### CODE REVIEW

**Review by kekey1 — DISMISSED** [review]

**Review by andlar — APPROVED** [review]

**Review by andlar — CHANGES_REQUESTED** [review]
PR has a merge conflict with v-next

**Review by andlar — DISMISSED** [review]

**Review by kekey1 — DISMISSED** [review]

File: changes/readme.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+## Need to add some directions on how the updated process works
```
Comment (andlar) [inline changes/readme.md]: Our other README files use all-caps for the file name. I'd like to keep that standard going forward

File: data-model-changes.sh [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+if [ -z "$HOST" ] || [ -z "$USER" ]
+then
+   echo "Some or all of the parameters are empty";
+   echo Host: $HOST
+   echo User: $USER
+   helpFunction
+fi
+
+#Exit on failure
+set -e
```
Comment (andlar) [inline data-model-changes.sh]: Can you move this to above any commands that happen in the script? If we're bombing out on failures I'd like to make sure we do it as early as the failure happens

File: data-model-changes.sh [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+while getopts 'h:u:?' flag; do
+    case "${flag}" in
+        h) HOST="${OPTARG}" ;;
+        u) USER="${OPTARG}" ;;
+        *) helpFunction ;;
+    esac
+done
+
+# Print helpFunction in case parameters are empty
+if [ -z "$HOST" ] || [ -z "$USER" ]
```
Comment (andlar) [inline data-model-changes.sh:28]: Can we default to `localhost` and `openchpl_dev` if they're empty? I think we do that in some of the scripts in the `dev/` directory, if you'd like to see a paradigm for doing that

**Review by andlar — CHANGES_REQUESTED** [review]

**Review by andlar — APPROVED** [review]

File: changes/README.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+## Need to add some directions on how the updated process works
+
```
Comment (kekey1) [inline changes/README.md:2]: Do you want to add those directions now?

Comment (tmy1313) [reply changes/README.md:2]: Talked to Andrew about this.  Someone can add the details when working on a subsequent PR.

**Review by kekey1 — APPROVED** [review]

**Pull request merged**